src/class-wp-import.php

```diff
@@ -708,9 +708,9 @@ protected function process_termmeta( $term, $term_id ) {
 		 * @param int   $term_id  ID of the newly created term.
 		 * @param array $term     Term data from the WXR import.
 		 */
-		$term['termmeta'] = (array) apply_filters( 'wp_import_term_meta', $term['termmeta'], $term_id, $term );
+		$term['termmeta'] = apply_filters( 'wp_import_term_meta', $term['termmeta'], $term_id, $term );
 
-		if ( empty( $term['termmeta'] ) ) {
+		if ( ! is_array( $term['meta'] ) || empty( $term['termmeta'] ) ) {
```

Comment:
:point_up: This is the one which causes the unit tests to fail. Then again, you may have intended the `is_array()` to check `$term['termmeta']`, not, as it does now, `$term['meta']`.